int16 number
float32 angle_min
float32 angle_max
float32 angle_inc
float32 time_inc
float32 scan_time
float32 range_min
float32 range_max
int16 range_size
int16 count
int16 countless
float32 lidar_ranges_1
float32 lidar_ranges_2
float32 lidar_ranges_3
float32 lidar_ranges_4
float32 lidar_ranges_5
float32 lidar_ranges_6
float32 lidar_ranges_7
float32 lidar_ranges_8
float32 lidar_ranges_9
float32 lidar_ranges_10
float32 lidar_ranges_11
float32 lidar_ranges_12
float32 lidar_ranges_13
float32 lidar_ranges_14
float32 lidar_ranges_15
float32 lidar_ranges_16
float32 lidar_ranges_17
float32 lidar_ranges_18
float32 lidar_ranges_19
float32 lidar_ranges_20
float32 lidar_ranges_21
float32 lidar_ranges_22
float32 lidar_ranges_23
float32 lidar_ranges_24

